tools/cli/admin_cluster_commands.go

```diff
@@ -36,7 +36,7 @@ import (
 
 // An indirection for the prompt function so that it can be mocked in the unit tests
 var promptFn = prompt
-var validSearchAttributeKey = regexp.MustCompile(`^[a-zA-Z][a-zA-Z_.-0-9]*$`)
+var validSearchAttributeKey = regexp.MustCompile(`^[a-zA-Z][a-zA-Z_0-9]*$`)
```

Comment:
same here. this regex should live close to other search attribute handling code in a central place.